test/benchmarks/material/tabs/app.module.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+import {MatTabsModule} from '@angular/material/tabs';
+
+@Component({
+  selector: 'app-root',
+  template: `
+    <button id="show-small-tab-group" (click)="showSmallTabGroup()">Show Small Tab Group</button>
+    <button id="show-large-tab-group" (click)="showLargeTabGroup()">Show Large Tab Group</button>
+    <button id="hide" (click)="hide()">Hide</button>
+
```

Comment:
I think that we also need a case which tests the tab group pagination since it triggers a bunch of layouts. I don't know if the example with 10 tabs below has enough content to trigger the pagination.

Reply:
What exactly would you like to test here? Should I try rendering like 100 tabs and clicking the mat-tab-header-pagination until it becomes disabled?

Reply:
Yeah that should work. Usually it's the act of measuring the header to figure out whether the pagination is supposed to be shown is the thing that triggers extra layouts.

Reply:
I settled for 20 tabs at the end because the tests were becoming a bit too large. At 100 tabs I even has jasmine time out on me. Let me know if you find this sufficient for gathering metrics on tab pagination.

Reply:
@crisbeto

Reply:
As long as it actually causes the pagination to show up it's enough.